### PR TITLE
refactor: use deno build script

### PR DESCRIPTION
**Pull request by uncenter**
Moves the preview images to where most repositories and the template have the images now, formats and cleans up some of the files (README, index.html), and switches to using a Deno build script which simplifies things a little bit.

### CODE REVIEW

**Comment by griimick** [comment]
Thanks! I am fine with moving assets and indentation fixes. I don't want to setup deno just to maintain this, would prefer we stick to nodejs for this. 

And why would you remove the comment though? 💀

**Comment by griimick** [comment]
@uncenter would you like to maintain this repo going forward? If yes, feel free to reopen this PR.

**Comment by uncenter** [comment]
Yeah I'd be down.

File: src/base.scss [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	--yellow: #{$yellow};
+	--aqua: #{$blue};
+	--orange: #{$peach};
+
+	--primary-bg: #{$base};
+	--current-line: #{$surface0};
+	--selection: #{$surface2};
+	--atom: #{$lavender};
+	--cursor: #{$overlay1};
+	--keyword: #{$red};
```
Comment (sgoudham) [inline src/base.scss:20]:
```suggestion
	--orange: #{$peach};

	--primary-bg: #{$base};
	--current-line: #{$surface0};
	--selection: #{$surface2};
	--atom: #{$lavender};
	--cursor: #{$overlay1};
	--keyword: #{$mauve};
```

**Review by sgoudham — CHANGES_REQUESTED** [review]
Just one thing I noticed to better match the style guide, I'm sure there's plenty of other keys that can be better matched but can get around to that later.

File: src/base.scss (hunk 1 above)
Comment (uncenter) [reply src/base.scss:20]: Yeah I was planning to update that as well. We've got all these variables set up for the GitHub userstyle too so we could change or reference from those.

Comment (uncenter) [reply src/base.scss:20]: 1. Maybe we'll leave that for a later PR with more updates?

Comment (sgoudham) [reply src/base.scss:20]: Yeah I just wanted to mention it more than anything

**Review by sgoudham — APPROVED** [review]

**Comment by sgoudham** [comment]
I'll transfer maintainer permissions now.

Thanks @griimick for getting this port off the ground in the first place :heart:

**Comment by griimick** [comment]
Thank you for all the things you do for the community. @sgoudham 

@uncenter Thank you for picking this up. I ported this for the logseq port which uses it as a git submodule. No need to worry about it, I plan to keep it locked to a specific commit.

**Pull request merged**